ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/recovery/TransactionStatusManager.java

```diff
@@ -85,8 +85,7 @@ private void addService( Service service, ServerSocket serverSocket )
          _listener = new Listener( serverSocket, service );
          _listener.setDaemon(true);
 
-	    tsLogger.i18NLogger.info_recovery_TransactionStatusManager_1(service.getClass().getName(),
-							  Integer.toString(serverSocket.getLocalPort()));
+         tsLogger.logger.debug(service.getClass().getName() + " starting");
```

Comment:
You have lost the service name in the debug message - the port number gets used in the thread name but I cannot see where we log this service name.